extension/src/popups/MinervaMenu.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undefined */
+window.onload = function() {
+    let studentNameField = document.getElementById("studentName");
+    studentNameField.onkeydown = function(event) {
+        if (event.code == "Enter" || event.key == "Enter") {
+            chrome.tabs.query({currentWindow: true, active: true}, function(tabs) {
```

Comment:
why do we need the `tabs` API? is there a reason we end up not using `tabs` variable?